<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderWindow</class>
 <widget class="QMainWindow" name="ReaderWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>688</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ReaderWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <widget class="QWidget" name="InputPage">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLineEdit" name="fileNameBox"/>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton">
              <property name="maximumSize">
               <size>
                <width>107</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Select File</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QComboBox" name="cutoffSelect">
              <property name="minimumSize">
               <size>
                <width>146</width>
                <height>0</height>
               </size>
              </property>
              <item>
               <property name="text">
                <string>Select Cutoff Type</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Median</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Median Multiple</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Percent of Max</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Absolute</string>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="cutoffAmount"/>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="AddFile">
            <property name="maximumSize">
             <size>
              <width>74</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Add</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QComboBox" name="averageCutoffType">
            <property name="minimumSize">
             <size>
              <width>152</width>
              <height>0</height>
             </size>
            </property>
            <item>
             <property name="text">
              <string>Average Cutoff Type</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Median</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Median Multiple</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Percent of Max</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Absolute</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="averageCutoff"/>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QScrollArea" name="scrollArea">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="FilesBox">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>658</width>
             <height>478</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_7">
            <item>
             <widget class="QWidget" name="fileSpacer" native="true">
              <layout class="QVBoxLayout" name="verticalLayout_8">
               <item>
                <spacer name="verticalSpacer_2">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>447</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Process</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="GraphPage">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QScrollArea" name="scrollArea_2">
          <property name="widgetResizable">
           <bool>true</bool>
          </property>
          <widget class="QWidget" name="scrollAreaWidgetContents">
           <property name="geometry">
            <rect>
             <x>0</x>
             <y>0</y>
             <width>214</width>
             <height>629</height>
            </rect>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QTextEdit" name="PeakList">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>190</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>2</number>
          </property>
          <property name="sizeConstraint">
           <enum>QLayout::SetDefaultConstraint</enum>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="legendToggle">
            <property name="maximumSize">
             <size>
              <width>120</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Toggle Legend</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>        O            C</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QScrollArea" name="VisibilityArea">
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="widgetResizable">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="VisibilityBox">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>0</y>
               <width>168</width>
               <height>544</height>
              </rect>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_6">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11"/>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="maximumSize">
             <size>
              <width>170</width>
              <height>16777215</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_13">
             <item>
              <widget class="QPushButton" name="HideAll">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Hide All</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="ShowAll">
               <property name="maximumSize">
                <size>
                 <width>60</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Show All</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QCustomPlot" name="plot" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <layout class="QHBoxLayout" name="horizontalLayout_3"/>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="saveGraph">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Width</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="graphWidth"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="graphHeight"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>File Type</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="graphFileType">
            <item>
             <property name="text">
              <string>Png</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Jpg</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Bmp</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>PDF</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButton_3">
            <property name="maximumSize">
             <size>
              <width>125</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionReset"/>
    <addaction name="actionGraph_Window"/>
    <addaction name="actionSave_Graph"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionSet_CI_Range"/>
    <addaction name="actionSet_EI_Range"/>
    <addaction name="actionCalculate_Ratio"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
  </widget>
  <action name="actionReset">
   <property name="text">
    <string>Main Window</string>
   </property>
  </action>
  <action name="actionGraph_Window">
   <property name="text">
    <string>Graph Window</string>
   </property>
  </action>
  <action name="actionSave_Graph">
   <property name="text">
    <string>Save Graph</string>
   </property>
  </action>
  <action name="actionSet_CI_Range">
   <property name="text">
    <string>Set CI Range</string>
   </property>
  </action>
  <action name="actionSet_EI_Range">
   <property name="text">
    <string>Set EI Range</string>
   </property>
  </action>
  <action name="actionCalculate_Ratio">
   <property name="text">
    <string>Calculate Ratio</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>actionReset</sender>
   <signal>triggered()</signal>
   <receiver>ReaderWindow</receiver>
   <slot>on_resetAction_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionGraph_Window</sender>
   <signal>triggered()</signal>
   <receiver>ReaderWindow</receiver>
   <slot>on_graphAction_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionSave_Graph</sender>
   <signal>triggered()</signal>
   <receiver>ReaderWindow</receiver>
   <slot>on_saveGraphButton_clicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>465</x>
     <y>339</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>on_resetAction_clicked()</slot>
  <slot>on_graphAction_clicked()</slot>
  <slot>on_saveGraphButton_clicked()</slot>
  <slot>on_About_clicked()</slot>
 </slots>
</ui>
